kernel/src/vm/vmar/vm_mapping.rs

```diff
@@ -46,6 +49,8 @@ pub(super) struct VmMapping {
     /// or are carried through to the underlying file for
     /// file-backed shared mappings.
     is_shared: bool,
+    /// Whether the mapping needs to handle surrounding pages when handling page fault.
+    need_handle_page_faults_around: bool,
```

Comment:
I suggest removing "need", just `handle_page_faults_around`. It is already an imperative and can be "do" for true and "don't" for false. Seems to be no other ambiguities found.